recipes/ozz-animation/all/test_package/CMakeLists.txt

```
@@ -1,8 +1,11 @@
 cmake_minimum_required(VERSION 2.8.11)
```

```suggestion
cmake_minimum_required(VERSION 3.1)
```

CXX_STANDARD requires CMake 3